### DISCUSSION TITLE
HLS video freenzing with normal audio

### DISCUSSION THREAD

**Discussion by silvafabio** [aclap-dev/video-downloadhelper#64] [Help and Q&A]
### Your question

Hi.

I'm downloading a serie of HLS vídeos and when it finished the video is freezing, but the audio is normal. 

I have tried with stable and beta versions.

Thanks

### Addon & CoApp version and details

version: 8.2.0.19
channel: beta
build date: 2024-02-08
build options: linuxlic: false, noyt: true, browser: chrome
lang: en-US
license: unneeded
platform: x86-64 linux
UA: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.17",
  "binary": "/home/fabio/vdhcoapp-2.0.17/vdhcoapp",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "linux",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "/home/fabio",
  "converterBinary": "/home/fabio/vdhcoapp-2.0.17/ffmpeg",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "63b936e"
}

**Comment by jcfrog** [comment]
hi,
is it a private site or is there a URL we can test?

**Reply by silvafabio** [reply]
Hi,

This is a private site :-(

I don't know if this information help to analyze, but i found a way right now: After download i use the converter "Re-encoded MPA (h264/aac)" and the audio and vídeo are perfect!.

This is a "normal" step or the tool should identify the right encode and download it without the conversion step?

Let me know if you need more information ;-)

Thanks
